provider/terraform/property_override.rb

```diff
@@ -133,15 +137,18 @@ def validate
 
         # Ensures boolean values are set to false if nil
         @sensitive ||= false
+        @is_set ||= false
 
         @default ||= Provider::Terraform::Default.new
 
         check_property :sensitive, :boolean
+        check_property :is_set, :boolean
         check_property :default, Provider::Terraform::Default
 
         check_optional_property :diff_suppress_func, String
         check_optional_property :state_func, String
         check_optional_property :validation, Provider::Terraform::Validation
+        check_optional_property :hash_func, String
```

Comment:
Different name here and above.  I prefer `set_hash_func`.

Reply:
Fixed.